src/Microsoft.CodeAnalysis.AnalyzerUtilities/PublicAPI.Unshipped.txt

```diff
@@ -719,7 +719,7 @@ virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TA
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.MergeAnalysisDataForBackEdge(TAnalysisData! value1, TAnalysisData! value2, Microsoft.CodeAnalysis.FlowAnalysis.BasicBlock! forBlock) -> TAnalysisData!
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.PostProcessArgument(Microsoft.CodeAnalysis.Operations.IArgumentOperation! operation, bool isEscaped) -> void
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.ProcessOutOfScopeLocalsAndFlowCaptures(System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.ILocalSymbol!>! locals, System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.FlowAnalysis.CaptureId>! flowCaptures) -> void
-virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.ProcessReturnValue(Microsoft.CodeAnalysis.IOperation? returnValueOperation) -> void
+virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.ProcessReturnValue(Microsoft.CodeAnalysis.IOperation? returnValue) -> void
```

Comment:
This update is done manually. The RS diagnostic didn't show locally for me in Visual Studio.